examples/05_stable_diffusion/src/compile_lib/compile_clip.py

```diff
@@ -23,14 +23,7 @@
 
 
 def map_clip_params(pt_mod, batch_size, seqlen, depth):
-
-    params_pt = list(pt_mod.named_parameters())
-
     params_ait = {}
-    pt_params = {}
-    for key, arr in params_pt:
-        pt_params[key.replace("text_model.", "")] = arr
```

Comment:
Why are we removing this replacement? Are we sure it's irrelevant? cc @terrychenism

Reply:
This particular `pt_params` is not used anywhere.
Right after this block of code we have another block of code which creates `pt_params` again and removes `text_model.` prefix 
```
    pt_params = dict(pt_mod.named_parameters())
    for key, arr in pt_params.items():
        name = key.replace("text_model.", "")
```